posthog/migrations/0782_remove_segment_hidden_destinations.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.22 on 2025-07-01 06:53
+
+from django.db import migrations
+
+
+def remove_segment_hidden_destinations(apps, schema_editor):
```

Comment:
Please consider making these separate PRs, especially if there is a migration completely unrelated to the other code

Reply:
Pull the migration into a separate PR: https://github.com/PostHog/posthog/pull/34388